schema.json

```diff
@@ -104,6 +104,7 @@
           "type": "string",
           "title": "Install-Path Schema",
           "description": "Folder where the application expects auto-installed/auto-updated APKs to be found. Default is value specified in 'target path'.",
+          "default": "",

```

Comment:
s there any json-schema way to actually tie this value to target-path as the description implies?